roachtest: activerecord failed

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=3113044&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=3113044&tab=artifacts#/activerecord) on master @ [95028269160e0bfa2b94477238e72670bf780e9d](https://github.com/cockroachdb/cockroach/commits/95028269160e0bfa2b94477238e72670bf780e9d):


```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/activerecord/run_1
	orm_helpers.go:245,orm_helpers.go:171,activerecord.go:233,test_runner.go:757: 
		Tests run on Cockroach v21.2.0-alpha.00000000-1750-g9502826916
		Tests run against activerecord 6.1
		6586 Total Tests Run
		6585 tests passed
		1 test failed
		18 tests skipped
		3 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: CockroachDB::ConnectionAdapters::PostgreSQLAdapterTest#test_database_exists_returns_false_when_the_database_does_not_exist (unexpected)
		--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
		For a full summary look at the activerecord artifacts 
		
		An updated blocklist (activeRecordBlockList21_2) is available in the artifacts' activerecord log
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
# From https://go.crdb.dev/p/roachstress, perhaps edited lightly.
caffeinate ./roachstress.sh activerecord
```
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #66622 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-21.1]
</p>
</details>
/cc @cockroachdb/sql-experience
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

--- a/pkg/cmd/roachtest/tests/activerecord.go
+++ b/pkg/cmd/roachtest/tests/activerecord.go
@@ -25,7 +25,7 @@ import (
 var activerecordResultRegex = regexp.MustCompile(`^(?P<test>[^\s]+#[^\s]+) = (?P<timing>\d+\.\d+ s) = (?P<result>.)$`)
 var railsReleaseTagRegex = regexp.MustCompile(`^v(?P<major>\d+)\.(?P<minor>\d+)\.(?P<point>\d+)\.?(?P<subpoint>\d*)$`)
 var supportedRailsVersion = "6.1"
-var activerecordAdapterVersion = "v6.1.2"
+var activerecordAdapterVersion = "v6.1.3"
 
 // This test runs activerecord's full test suite against a single cockroach node.
 

